openslides_backend/action/action.py

```diff
@@ -195,7 +195,7 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
                     meeting_id,
                 ):
                     return
-                msg_appendix = f" Missing permission: {self.permission}"
+                raise MissingPermission(self.permission)
```

Comment:
You can also throw a `MissingPermission` in the above case for OML.

Reply:
Changed.